sdk/storage/inc/blobs/internal/protocol/blob_rest_client.hpp

```diff
@@ -831,8 +830,14 @@ namespace Azure { namespace Storage { namespace Blobs {
       {
         auto request = Azure::Core::Http::Request(Azure::Core::Http::HttpMethod::Get, url);
         request.AddHeader("Content-Length", "0");
-        request.AddHeader("x-ms-version", options.Version);
-        request.AddHeader("x-ms-date", options.Date);
+        if (!options.Version.empty())
+        {
+          request.AddHeader("x-ms-version", options.Version);
+        }
+        else
+        {
+          request.AddHeader("x-ms-version", "2019-07-07");
```

Comment:
>2019-07-07 [](start = 45, length = 10)

Please raise this topic during internal discussion meeting: What version should we be targeting for private preview for blob service.
BTW I don't think 2019-07-07 is a public version right? we have 2019-02-02 and 2019-12-12?

Reply:
2019-07-07 is public, 2019-10-10 is not public